manifests/0.5.1/3scale-community-operator.v0.5.1.clusterserviceversion.yaml

```diff
@@ -115,7 +127,7 @@ spec:
     * **Capabilities** Ability to define 3scale API definitions and set them into a 3scale API Management solution
 
     ### Documentation
-    Documentation can be found on our [website](https://github.com/3scale/3scale-operator/blob/v0.5.0/doc/operator-user-guide.md).
+    Documentation can be found on our [website](https://github.com/3scale/3scale-operator/blob/2.8-stable/doc/operator-user-guide.md).
```

Comment:
This should be 3scale-2.8-stable instead of 2.8-stable  (later is getting deprecated)

Reply:
:+1: